### DISCUSSION TITLE
All profile README titles are now blue

### DISCUSSION THREAD

**Discussion by bokub** [community/community#70635] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello

All the links from my profile README are now **blue** instead of black.

This looks like a bug that only affects **profile READMEs** and **not** repository READMEs.

## Screenshot & CSS
![image](https://github.com/community/community/assets/17952318/62abd8b1-828b-4da5-85d0-6d7abd1725f1)

**Comment by Shariar-Hasan** [comment]
its because, when you create a heading in Markdown, like this:

```markdown
## My Heading
```

Some platforms (like gihube) automatically generate an anchor link based on the heading text. For example, the anchor link for the heading above might look something like this:

```html
<a href="#my-heading">My Heading</a>
```

This creates a link to the heading within the same document. When you click on this link, it scrolls the page to the heading.

This behavior can be very useful and it not a bug 🙂

**Reply by bokub** [reply]
The bug is not on the links being present, but on the blue color. I just changed my issue title as it was confusing.

Exemple on a repository README, where the anchors are black:

![image](https://github.com/community/community/assets/17952318/b9481cdc-7f6f-40cd-ad16-0db0f010ae2a)

**Reply by Shariar-Hasan** [reply]
Well in that case it might be a bug. or might not 🙂

**Comment by spenserblack** [comment]
Just a guess, but this might be related to the latest styling change for links: #68734. Perhaps they (intentionally or not) affected the styling for headers when working on this accessibility option.

**Comment by vlakoff** [comment]
Agree, these are not navigation links, and should not be colored.

I was about to suggest that when these anchor links are hovered, a « § » symbol should be displayed next to the text (having a reserved margin for this symbol, so that no line break or overflow happens when the symbol appears).

But apparently, that was already the case, but it is currently broken. See after the links, there are `<svg class="octicon octicon-link">` elements, that never appear currently.
